test/conftest.py

```diff
@@ -0,0 +1,74 @@
+"""Pytest configuration and fixtures for uxarray tests."""
+
+import pytest
+import numpy as np
+from pathlib import Path
+
+
+@pytest.fixture
+def gridpath():
+    """Get the path to a test grid/mesh file.
+
+    Parameters
+    ----------
+    *args : str
+        Path components relative to test/meshfiles/
+
+    Returns
+    -------
+    Path
+        Full path to the test grid file
+    """
+    base_path = Path(__file__).parent / "meshfiles"
+
+    def _get_path(*args):
+        path = base_path.joinpath(*args)
+        if not path.exists():
+            pytest.skip(f"Test grid file not found: {path}")
+        return path
+
+    return _get_path
+
+
+@pytest.fixture
+def datasetpath():
```

Comment:
Could we extend this to support returning a list of data paths if the user provides multiple, such as for the geoflow case?

Reply:
Yes, that can be done datasetpath("ugrid", "geoflow-small", ["v1.nc", "v2.nc", "v3.nc"]) would return a list of paths